src/etl/adapter/loader.py

```
@@ -0,0 +1,39 @@
+
+from src.etl.domain.value_objects import MessageMetadata
+
+class IDataLoader(ABC):
+
+    @abstractmethod
+    async def load(self, source: str) -> List[MessageMetadata]:
+        pass
+
+class TelegramLoader(IDataLoader):
+
+    def __init__(self, grabber: TelegramGrabber):
```

Ruff (0.15.12)

[error] Undefined name `TelegramGrabber`

(F821)